### DISCUSSION TITLE
5.38.0.1 - readline-8.2 not included?

### DISCUSSION THREAD

**Discussion by mikecappella** [StrawberryPerl/Perl-Dist-Strawberry#161] [General]
Sorry if I've missed some post regarding this.  It seems 5.38.0.1 is not including the readline-8.2 library.

Is this intentional?

**Comment by shawnlaffan** [comment]
libreadline was included with 5.32 but was missed in the 5.36/5.38 build process.  It is part of the 5.36 and 5.38 PDL editions, though.  

It can also be downloaded from https://github.com/StrawberryPerl/build-extlibs/releases/tag/dev_gcc13.1_20230606.  Look for 64bit_readline-8.2-bin_20230606.zip

**Comment by mikecappella** [comment]
Thanks for the info.

I discovered this issue after one of my software user's informed me that my modules installer script was failing due to being unable to build Term::Readline::GNU.

**Comment by shawnlaffan** [comment]
It will be in 5.38.2.2, which is currently being built.

**Comment by shawnlaffan** [comment]
5.38.2.2 is at https://github.com/StrawberryPerl/Perl-Dist-Strawberry/releases/tag/SP_53822_64bit
